doc/locale/ja/LC_MESSAGES/reference/commands/select.po

```diff
@@ -3185,6 +3187,46 @@ msgstr ""
 "ません。この場合は ``_key`` を使ってください。これは、 :ref:`select-"
 "drilldown-output-columns` と同じルールです。"
 
+msgid ""
+"Specifies ``${LABLE}`` of other ``drilldown``, ``drilldowns`` or ``slices``."
+msgstr ""
+"他の ``drilldown``、 ``drilldowns`` または ``slices`` の ``${LABLE}`` を指定"
+"します。"
+
+msgid ""
+"The result of the specified ``${LABLE}`` is drilldowned, which means that "
+"this paramter enables nested aggregate calculations and groups in drilldown."
+msgstr ""
+"指定した ``${LABLE}`` の結果をドリルダウンします。つまり、このパラメータは多"
+"段ドリルダウンを可能にします。"
+
+msgid ""
+"Here is an example to execute nested drilldown by tag and then by category."
+msgstr "以下は、タグ、カテゴリーの順で多段ドリルダウンを行う場合の例です。"
+
+msgid ""
+"In this example, the schema contains the table named as ``Memo`` which has "
```

Comment:
```suggestion
In this example; 
The schema contains the table named as ``Memo`` which has the column named as ``tag``.
the table named as ``Tags`` which has the columns named as ``category``.
```
when it comes to line up fact, breaking up sentence would be easier to understand.